packages/ember-cookies/package.json

```diff
@@ -43,6 +43,7 @@
     "@babel/plugin-proposal-decorators": "7.23.0",
     "@embroider/addon-dev": "4.1.1",
     "@rollup/plugin-babel": "6.0.4",
+    "@types/ember": "^4.0.6",
```

Comment:
It seems like this dependency is extraneous, since the `index.d.ts` is not extending any of the Ember types?

Reply:
Thanks for the comment make sense. I will update the pr